File: RQ1_Prompt_Elements/data/scenario1/id_77.java
package scenario1;


class Iscube {

    public static Boolean iscube(int a) {
        if (a < 0) {
            return iscube(-a);
        }
        int i = 0;
        while (i * i * i < a) {
            i++;
        }
        return i * i * i == a;
    }

}

File: HumanEvalPython/scenario2/id_77.py

def iscube(a):
    '''
    Write a function that takes an integer a and returns True 
    if this ingeger is a cube of some integer number.
    Note: you may assume the input is always valid.
    
    '''

    a = abs(a)
    return int(round(a ** (1. / 3))) ** 3 == a
